common4j/src/main/com/microsoft/identity/common/java/providers/nativeauth/responses/signup/SignUpContinueApiResponse.kt

```diff
@@ -0,0 +1,179 @@
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.common.java.providers.nativeauth.responses.signup
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+import com.microsoft.identity.common.java.logging.LogSession
+import com.microsoft.identity.common.java.providers.nativeauth.IApiResponse
+import com.microsoft.identity.common.java.providers.nativeauth.responses.UserAttributeApiResult
+import com.microsoft.identity.common.java.util.isAttributeValidationFailed
+import com.microsoft.identity.common.java.util.isAttributesRequired
+import com.microsoft.identity.common.java.util.isCredentialRequired
+import com.microsoft.identity.common.java.util.isExpiredToken
+import com.microsoft.identity.common.java.util.isInvalidOOBValue
+import com.microsoft.identity.common.java.util.isInvalidRequest
+import com.microsoft.identity.common.java.util.isOtpCodeIncorrect
+import com.microsoft.identity.common.java.util.isPasswordBanned
+import com.microsoft.identity.common.java.util.isPasswordRecentlyUsed
+import com.microsoft.identity.common.java.util.isPasswordTooLong
+import com.microsoft.identity.common.java.util.isPasswordTooShort
+import com.microsoft.identity.common.java.util.isPasswordTooWeak
+import com.microsoft.identity.common.java.util.isUserAlreadyExists
+import com.microsoft.identity.common.java.util.isVerificationRequired
+import com.microsoft.identity.common.java.util.toAttributeList
+import java.net.HttpURLConnection
+
+/**
+ * Represents the raw response from the Sign Up /continue endpoint.
+ * Can be converted to SignUpContinueApiResult using the provided toResult() method.
+ */
+data class SignUpContinueApiResponse(
+    @Expose override var statusCode: Int,
+    @SerializedName("signin_slt") val signInSLT: String?,
+    @Expose @SerializedName("expires_in") val expiresIn: Int?,
+    @Expose @SerializedName("error") val error: String?,
+    @Expose @SerializedName("error_codes") val errorCodes: List<Int>?,
+    @Expose @SerializedName("error_description") val errorDescription: String?,
+    @SerializedName("signup_token") val signupToken: String?,
+    @Expose @SerializedName("unverified_attributes") val unverifiedAttributes: List<Map<String, String>>?,
+    @Expose @SerializedName("invalid_attributes") val invalidAttributes: List<Map<String, String>>?,
+    @Expose @SerializedName("required_attributes") val requiredAttributes: List<UserAttributeApiResult>?,
+    @Expose @SerializedName("details") val details: List<Map<String, String>>?
```

Comment:
Do we know if "details" will ever contain PII?

Reply:
I have removed the exposed attribute on details field in all responses .